examples/community/run_pixart_alpha_controlnet_pipeline.py

```diff
@@ -0,0 +1,81 @@
+import sys
```

Comment:
this should go into README https://github.com/huggingface/diffusers/blob/main/examples/community/README.md

Reply:
i have added the section

Reply:
this file should not be here, no? should be in the same folder as others?

Reply:
just moved it